lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -322,7 +322,7 @@ function processBucketPolicy(requestType, bucket, canonicalID, arn, bucketOwner,
 }
 
 function isBucketAuthorized(bucket, requestTypesInput, canonicalID, authInfo, log, request,
-    actionImplicitDeniesInput = {}) {
+    actionImplicitDeniesInput = {}, isWebsite = false) {
```

Comment:
Echoing your comment around the number of args here, another way could be to have a last parameter as an object, where we can set various properties depending on the needs, it would allow us to reuse if we need more specific logic, but not a blocker for this PR